src/featureforest/utils/downloader.py

```diff
@@ -0,0 +1,45 @@
+from pathlib import Path
+import pooch
+
+
+MODELS_CACHE_DIR = Path.home().joinpath(".featureforest").joinpath("models")
+MODELS_CACHE_DIR.mkdir(parents=True, exist_ok=True)
```

Comment:
I believe pooch takes care of the directory and file creation automatically

Reply:
Redundant codes were removed.